frontend/.eslintrc.js

```diff
@@ -1,13 +1,11 @@
 "use strict";
 
 module.exports = {
-  extends: "@adfinis-sygroup/eslint-config/ember-app",
-
+  settings: {
+    "import/internal-regex": "^customer-center/",
+  },
+  extends: ["@adfinis/eslint-config/ember-addon"],
   rules: {
-    "no-unused-vars": "off",
-    "prefer-rest-params": "off",
-    "ember/use-brace-expansion": "off",
-    "ember/no-get": "off",
-    "ember/no-mixins": "warn",
```

Comment:
I think we still need them.

Reply:
Re-added as of https://github.com/adfinis/customer-center/pull/412/commits/179426966a197d6f2126c01d5767b25b74542d4f